android/app/src/androidTest/java/org/ZingoLabs/Zingo/BobaTest.kt

```diff
@@ -0,0 +1,68 @@
+package org.ZingoLabs.Zingo
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.experimental.categories.Category
+
+@Category(IntegrationTest::class)
+class BobaTest {
+
+    data class Addresses (
+    	val address : String,
+    	val receivers : Receivers
+    )
+
+    data class Receivers (
+    	val transparent : String,
+    	val sapling : String,
+    	val orchard_exists : Boolean
+    )
+
+    @Test
+    fun BobaTest() {  
+        val server = "https://testnet.lightwalletd.com:9067"
+        val seed = "very airport kite ordinary pelican there like spider kitchen top meat zone subway web recycle original merge key depth popular broom snack gaze stumble"
+        val birthday = "2311722"
+        val datadir = MainApplication.getAppContext()!!.filesDir.path
+
+        RustFFI.initfromseed(server, seed, birthday, datadir)
+
+        var balance = RustFFI.execute("balance", "")
+        System.out.println("Balance:")
```

Comment:
unfortunately i cant find these log messages from the test in `android/app/build/outputs/integration_test_reports

Reply:
Try grepping the logcat.txt for 'system.out' I put the exact command I use in chat couple days ago

Reply:
i ussually ```cd``` into the ```android/app/build/outputs/integration_test_reports``` directory
then i set this up in a terminal while im running tests:
```watch -n 1 'cat x86_64/logcat.txt | grep 'System.out' | tail -50'```